### DISCUSSION TITLE
is there any way to list EXTERNAL IDENTITY PROVIDERS of a user?

### DISCUSSION THREAD

**Discussion by ranjiteds** [zitadel/zitadel#6187] [Q&A] [answered]
I want to list all the EXTERNAL IDENTITY PROVIDERS of a user with EXTERNAL USER ID, trying to look at the APIs but could not find this. any suggestions on this.

**Comment by ranjiteds** [comment, marked as answer]
https://zitadel.com/docs/apis/resources/auth/auth-service-list-my-linked-id-ps

**Reply by fforootd** [reply]
Thanks for sharing the resolution.
Let us know if you have more questions.
